app/Repositories/SyllableRepository.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repositories;
+
+use App\Entities\SelectedSyllable;
+use App\Entities\Syllable;
+use PDO;
+
+class SyllableRepository
+{
+    public function __construct(
+        private readonly PDO $connection
+    ){
+    }
+
+    public function loadSyllablesFromFileToDb(string $fileName): void
+    {
+        $path = dirname(__DIR__) . $fileName;
+
+        if (!file_exists($path)) {
+            throw new \InvalidArgumentException("File not found with path {$path} and file name {$fileName}!");
+        }
+
+        try {
+            $stmt = $this->connection->prepare(
+                "LOAD DATA LOCAL INFILE :path INTO TABLE syllables
+                FIELDS TERMINATED BY ''
+                (pattern)"
+            );
+
+            $stmt->execute(['path' => $path]);
+        } catch (PDOException $e) {
+            echo $e->getMessage();
```

Comment:
With this line repository is now responsable for two things - output and doing repository stuff. So if this is necessary for user to see (which I doubt) use 
a) Exceptions, and handle them on higher level, for handler use IOService 
b) Use IOService here (dirtier solution)

If this info is not needed for user and only for developer then loghger should be used here
a) Same as before just use logger
b) Same as before but not considered as "dirty" solution (still a bit dirtier 😄)